packages/foreman/katello-certs-tools/katello-certs-tools.spec

```diff
@@ -2,40 +2,63 @@ Name:      katello-certs-tools
 Summary:   Katello SSL Key/Cert Tool
 Group:     Applications/Internet
 License:   GPLv2
-Version:   2.4.0
+Version:   2.5.0
 Release:   1%{?dist}
 URL:       https://github.com/katello/katello-certs-tools
-Source0:   https://codeload.github.com/Katello/%{name}/tar.gz/%{version}
+Source0:   https://codeload.github.com/Katello/%{name}/tar.gz/%{version}#/%{name}-%{version}.tar.gz
 BuildArch: noarch
 
-#BZ 1022017
-Requires: openssl >= 1.0.1e-16
-
+Requires: openssl
 Requires: rpm-build
 
 BuildRequires: docbook-utils
-BuildRequires: python
+BuildRequires: /usr/bin/pathfix.py
+%if 0%{?rhel} > 7
```

Comment:
I'm wondering about reversing the logic and only build with py2 if it's rhel 7. That way we also support Fedora. Not a blocker though.